spring-vault-core/src/main/java/org/springframework/vault/core/ReactiveKeyValueHelper.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2018-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.vault.core;
+
+import org.springframework.vault.support.VaultResponseSupport;
+import reactor.core.publisher.Mono;
+
+/**
+ * Helper for wrapping imperative operations.
+ *
+ * @author Timothy R. Weiand
+ * @since 3.999.999
```

Comment:
The target version will be `3.1`, so `@since 3.1` is the way to go across all newly introduced classes and methods.